Documentatie/EP-online-validaties.md

```diff
@@ -1,114 +1,98 @@
 # Businessrules registratiebestand EP-online
 
-**Versiebeheer**
-
-| Datum | Versie  | Auteur  | Wijzigingen
-| ------------:  |	:------------:	|	:------------	|	:-----------------------------------------------
-| 6 aug 2019	|	0.1	|	Peter Zaal	|	Initiële versie.
-|	|	0.2	|	Peter Zaal	|	Foutjes eruit gehaald.
-|	9 sep 2019	|	0.3	|	Peter Zaal	|	Bijgewerkt met uitgesplitste SurveyDate rules.
+## 1. Versiebeheer
+| Datum 		| Versie| Auteur  			| Wijzigingen
+|--------------:|:-----:|:-----------------:|:----------------------------------------------------------------------------
+|	6 aug 2019	|	0.1	|	Peter Zaal		|	Initiële versie.
+|				|	0.2	|	Peter Zaal		|	Foutjes eruit gehaald.
+|	9 sep 2019	|	0.3	|	Peter Zaal		|	Bijgewerkt met uitgesplitste SurveyDate rules.
 |	12 sep 2019	|	0.4	|	Piet Vredeveld	|	Omgezet naar Markdown formaat, EP-online interne informatie verwijderd.
-|	24 jan 2020 |	0.5 |	Paul Kamps		|	Nieuwe validaties en checks toegevoegd.
-| 07 jul 2020 | 0.6 | Bas Brouwer  | Geüpdatet aan de hand van de nieuwe XSD versie.
+|	24 jan 2020	|	0.5 |	Paul Kamps		|	Nieuwe validaties en checks toegevoegd.
+|	07 jul 2020	|	0.6 |	Bas Brouwer  	| 	Geüpdatet aan de hand van de nieuwe XSD versie.
+|	10 jul 2020	|	0.6 |	Paul Kamps  	| 	Business Rules geüpdatet en toegevoegd.
 
-## Validaties
+## 2. Validaties
 Het valideren en verwerken van het registratiebestand gebeurt in een aantal stappen. Als er een of meerdere validaties in een stap niet voldoen, worden de betreffende bijbehorende meldingen gegeven en niet verder gegaan naar de volgende stap.
 
-## 1. Bepalen versienummer en rekenmethodiek
-|	Technische naam	|	Rule(s)
-|	:------------------------	| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-|	CalculationType	|	TypeCalculation, Versie, MainBuildingClass en Opnametype worden uitgelezen. Aan de hand hiervan wordt bepaald wat de rekenmethodiek is. De rekenmethodiek moet bestaan in EP-online.
-|	CalculationTypeVersion	|	De XSD versie bij de rekenmethodiek moet bestaan in EP-online.
-|	MonitoringVersion	|	Element *EPMeta.Version* moet bestaan en een geldig getal zijn; dit bepaalt de monitorbestandversie.
-|	GenericXmlVersion	|	In de generieke XML moet het veld *Energieprestatie.Versie* gevuld zijn, een geldig getal zijn, en overeenkomen met de generieke XML versie in EP-online.
+### 2.1. Ophalen en valideren gebruiker (CH) en organisatie
+Gegevens omtrent de CH en organisatie worden opgehaald. Er vinden validaties plaats dat de gegevens gevonden kunnen worden. Indien dit niet lukt, wordt er geen functionele foutmelding gegeven, maar treed er een technische fout op. Al deze gegevens zijn al eerder gevalideerd dus moeten nog steeds bestaan.
```

Comment:
"deze gegevens zijn al eerder gevalideerd" en "treed er een technische fout op". Dit lijkt mij interne informatie en niet relevant/nodig voor het publieke document.

Graag voor interne document veranderen naar: "De gegevens omtrent de CH en de organisatie worden opgehaald. Vervolgens vinden er validaties plaats of de gegevens zijn gevonden. Indien dit niet lukt, wordt er geen functionele foutmelding gegeven, maar treed er een technische fout op. Aangezien deze gegevens al eerder _[in het process?]_ gevalideerd zijn, kan dit niet fout gaan."